explorer/e2e/testInterfaces.ts

```diff
@@ -49,14 +50,27 @@ export interface BackpageAccessTags {
   grantedShortName: string;
 }
 
-export interface BackpageExportButtons {
-  accessNotGrantedMessage: string;
+export interface ExportButtonInfo {
   detailsName: string;
   exportActionButtonText: string;
   exportRequestButtonText: string;
+  firstLandingMessage?: string; //TODO: rename to requestLoadingMessage
```

Comment:
Two q's here (and for `secondLandingMessage`):

1. Do you want to go ahead and do the rename in this PR?
2. Why is this optional? It looks like it might be causes side effects downstream (e.g. `?? ""` [here](https://github.com/DataBiosphere/data-browser/pull/4217/files#diff-faefc5cfc7d7c7283d514ad866678143558aefea3eccb636b8bb7a1ae4abbae7R1083))?

Reply:
Hi! This one was optional because the details page tests initially didn't use each landing message, and I didn't want to make changes there. I've gone ahead and updated those tests so they all check landing messages but do not check loading messages, since there was an issue with loading messages not being detected by Playwright and leading to flakiness. Let me know if that solution works!